src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/ClanManager.java

```diff
@@ -224,7 +225,7 @@ public ClanPlayer getClanPlayerJoinEvent(Player player) {
      * @param player
      * @return
      */
-    public ClanPlayer getClanPlayer(Player player) {
+    public ClanPlayer getClanPlayer(OfflinePlayer player) {
```

Comment:
Should've added a helper method here accepting a Player object and passing to this method, you just broke a ton of plugins that use this method.

Reply:
Is not it possible to pass a Player as an OfflinePlayer? Player extends OfflinePlayer...

Reply:
Plugins that import older versions of SimpleClans will fail because it's trying to invoke a method that accepts a Player argument, not an OfflinePlayer.

(i.e. it's irrelevant whether you can pass a normal Player as an OfflinePlayer).

Reply:
This is what happens:
```
java.lang.NoSuchMethodError: net.sacredlabyrinth.phaed.simpleclans.managers.ClanManager.getClanPlayer(Lorg/bukkit/entity/Player;)Lnet/sacredlabyrinth/phaed/simpleclans/ClanPlayer;
	at me.robomwm.MountainDewritoes.SimpleClansListener.setDisplayName(SimpleClansListener.java:143) ~[?:?]
	at me.robomwm.MountainDewritoes.SimpleClansListener.onPlayerJoin(SimpleClansListener.java:80) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1417.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.12.2.jar:git-Paper-1296]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1296]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1296]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:515) [patched_1.12.2.jar:git-Paper-1296]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:500) [patched_1.12.2.jar:git-Paper-1296]
	at net.minecraft.server.v1_12_R1.PlayerList.onPlayerJoin(PlayerList.java:370) [patched_1.12.2.jar:git-Paper-1296]
	at net.minecraft.server.v1_12_R1.PlayerList.a(PlayerList.java:183) [patched_1.12.2.jar:git-Paper-1296]
	at protocolsupport.zplatform.impl.spigot.network.handler.SpigotLoginListenerPlay.joinGame(SpigotLoginListenerPlay.java:124) [ProtocolSupport.jar:?]
	at protocolsupport.protocol.packet.handler.AbstractLoginListenerPlay.tryJoin(AbstractLoginListenerPlay.java:132) [ProtocolSupport.jar:?]
	at protocolsupport.protocol.packet.handler.AbstractLoginListenerPlay.tick(AbstractLoginListenerPlay.java:94) [ProtocolSupport.jar:?]
	at protocolsupport.zplatform.impl.spigot.network.handler.SpigotLoginListenerPlay.e(SpigotLoginListenerPlay.java:70) [ProtocolSupport.jar:?]
	at net.minecraft.server.v1_12_R1.NetworkManager.a(NetworkManager.java:254) [patched_1.12.2.jar:git-Paper-1296]
	at net.minecraft.server.v1_12_R1.ServerConnection.c(ServerConnection.java:150) [patched_1.12.2.jar:git-Paper-1296]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:940) [patched_1.12.2.jar:git-Paper-1296]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:426) [patched_1.12.2.jar:git-Paper-1296]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:767) [patched_1.12.2.jar:git-Paper-1296]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:665) [patched_1.12.2.jar:git-Paper-1296]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
```